rclpy/test/test_messages.py

```diff
@@ -47,7 +47,14 @@ def test_unicode_string(self):
         self.node.destroy_publisher(pub)
 
     def test_different_type_raises(self):
-        pub = self.node.create_publisher(BasicTypes, 'chatter', 1)
+        # TODO(bmarchi): When a publisher is destroyed for opensplice,
```

Comment:
What is the action items of the TODO?

Reply:
Revert the topic name again to chatter when proper destruction can be done. I'll add that